tests/Integration/Behaviour/Features/Context/Domain/CartRuleFeatureContext.php

```diff
@@ -211,7 +209,7 @@ public function specifyCartRuleGivesPercentageReduction(
         $discountApplicationType
     ) {
         $properties['reduction_percentage'] = (float) $percentage;
-        $properties['reduction_applies_to_discounted_products'] = $includesDiscountedProducts;
```

Comment:
I still think this naming makes more sense

Reply:
![image](https://user-images.githubusercontent.com/31609858/230574710-261a3142-1c14-40e4-932e-10bc4be07cc8.png)
I'll make it "exclude_discounted_products" as it is in UI and in object model is called `reduction_exclude_special`, the most important thing to get rid of that double negation everywhere :sweat:

Reply:
Renamed in UI and command to `Apply to discounted products`